pkg/services/accesscontrol/resourcepermissions/options.go

```diff
@@ -18,6 +18,8 @@ type Options struct {
 	ResourceAttribute string
 	// OnlyManaged will tell the service to return all permissions if set to false and only managed permissions if set to true
 	OnlyManaged bool
+	// OnlyActionSets will tell the service to only write and read actionsets
```

Comment:
Writing is the main thing - we will read whatever is written in the DB, but what we want is to test how RBAC performs if we don't write fine-grained permissions to the DB.

Reply:
no longer needed